### PR TITLE
Added styling to evaluation page

### LINKED ISSUE #179: Evaluation page could use some visual TLC
This is one of the first interactions new potential customers have experience with Insights.

No changes to the evaluation logic itself, just the visual presentation of the page.
access.redhat.com/insights/evaluation

### PR DESCRIPTION
**Pull request by ryelo**
Fixes #179 

Will revisit once @dkuc gets back to update the error page.

### CODE REVIEW

**Comment by Kinlaw** [comment]
@ryelo is this ready to be reviewed?

**Comment by ryelo** [comment]
@Kinlaw Yessir!

**Comment by iphands** [comment]
@Kinlaw anything blocking this?

File: app/js/states/evaluation/evaluation.jade [hunk 1]
```diff
@@ -1,84 +1,177 @@
...
-          .col-sm-12.col-md-8.col-md-offset-2
-            h1(translate)
-              span.icon-check-alt(aria-hidden="true")
-              | &nbsp;Thank you for purchasing Red Hat Insights.
+        .status-current(ng-if='status.current')
+          .row.text-center
+            .col-sm-12.col-md-8.col-md-offset-2
+              h1.eval-header(translate) You are currently on a {{status.current.systems}} system evaluation of Red Hat Insights
+              hr.eval-hr
+              h2.eval-subhead Your evaluation will expire on
```
Comment (Kinlaw) [inline app/js/states/evaluation/evaluation.jade]: missing translate tag

Comment (ryelo) [reply app/js/states/evaluation/evaluation.jade]: Updated!

**Pull request merged**